.github/workflows/test.yaml

```diff
@@ -31,23 +31,20 @@ jobs:
         strategy:
             matrix:
                 php-version:
-                    - '8.1'
-                    - '8.2'
-                    - '8.3'
+                    - '8.4'
```

Comment:
I removed all the PHP versions from the matrix, we only need to test the latest and lowest supported versions of PHP for each Symfony version.